.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-.env
\ No newline at end of file
+.env
+.DS_Store
```

Comment:
@claudebarde I suggest keeping an empty line at the end of all files. If we do it, adding new content only changes the added lines.
If we don't, we get an extra newline character at the end of last line.

This is a minor nuisance when doing code review, but a bit of more serious one when checking out the history of a line (or when your IDE shows you the last person and commit when a line is changed). They will all miss the real one.

Same reason for adding commas to last items in lists (when items are on separate lines) and other separators that are not necessary for the last item.

Reply:
Also, your `editorconfig` file can be set to add that automatically.